content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -282,98 +287,109 @@ This example shows the data contained in an `AdmissionReview` object
 for a request to update the `scale` subresource of an `apps/v1` `Deployment`:
```

Comment:
This was written in YAML because it allows comments.

I'm wary to drop the comments, and JSON doesn't have comments.
Switching it to JSON-like YAML feels reasonable.